examples/02-HFSS/Array.py

```diff
@@ -148,10 +148,6 @@
 # Generate 3D polar plots in Matplotlib. You can define
 # the Theta scan and Phi scan.
 
-plot_3d = ffdata.plot_3d(quantity='RealizedGain', image_path=os.path.join(working_directory, "Image.jpg"),  show=False)
-
-##########################################################
-# Close all matplotlib figures
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-plt.close('all')
+# plot_3d = ffdata.plot_3d(quantity='RealizedGain',
+#                          output_file=os.path.join(working_directory, "Image.jpg"),
+#                          show=False)
```

Comment:
Since this is a temporary comment, can you create an issue to track this problem later ?